packages/core/src/browser/status-bar/status-bar-elements.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2017-2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as React from "react";
+
+export class StatusBarElements extends React.Component<StatusBarElements.Props> {
+    render() {
+        return <div key={"statusbar-elements-" + this.props.alignment + "-area"} className={"area " + this.props.alignment}>{this.props.entries}</div>;
```

Comment:
fyi: keys are only making sense into an array, here you have the following structure:
`<StatusBarElements><div key="..." /></StatusBarElements>` - key is redundant as you can see

Generally when you use jsx keys are not necessary, they are needed only then jsx elements are collected into an array and only for direct members of the array